### PR TITLE
create Min_Max_update_query.cpp

### LINKED ISSUE #57: Segment tree for updating and finding min-max in given Range in an array.
Can I take this @mayank64ce

### PR DESCRIPTION
**Pull request by thepritam**
Adding min max and update query in an array using segment tree

Fixes #57

### CODE REVIEW

**Comment by thepritam** [comment]
@s-ayush2903  please look into this.

File: Min_Max_update_query.cpp [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+using vl = vector<ll>;
+using vd = vector<db>; 
+using vs = vector<str>;
+using vpi = vector<pi>;
+using vpl = vector<pl>; 
+using vpd = vector<pd>;
+ 
+
+#define mp make_pair
+#define f first
```
Comment (s-ayush2903) [inline Min_Max_update_query.cpp:33]: Please remove all the unused parameters

Comment (thepritam) [reply Min_Max_update_query.cpp:33]: sure i will do it.

**Comment by thepritam** [comment]
please check the above code @s-ayush2903

**Comment by thepritam** [comment]
I have created a new pull request for this .I am closing this Pull request.

**Pull request closed without merging**